src/command/change.go

```diff
@@ -16,14 +16,18 @@ import (
 // of a specific application on the fly.
 type ChangeCommand struct {
 	UI       ui.UI
-	Projects api.ProjectsUpdater
-	Apps     api.AppsUpdater
+	Projects interface {
+		api.ProjectsUpdater
+		api.ProjectsGetter
+		api.ProjectsCreater
+	}
+	Apps api.AppsUpdater
 }
 
 // Help should return long-form help text.
 func (c *ChangeCommand) Help() string {
 	helpText := `
-Usage: sloppy change [OPTIONS] (PROJECT/SERVICE/APP | PROJECT FILENAME)
+Usage: sloppy change [OPTIONS] (PROJECT/SERVICE/APP | FILENAME)
```

Comment:
Can we make sure the old syntax (sloppy change PROJECT FILENAME) still works after this change? Or at least throw an explicit error about the change? If the latter, we should still do a major version bump.